app/javascript/src/utils/dateFormats.js

```diff
@@ -0,0 +1,7 @@
+import dayjs from "dayjs";
```

Comment:
https://academy.bigbinary.com/learn-react/pages/structuring-code#:~:text=Keep%20the%20files%20in%20closest%20scope

Also the filename `dateFormats` is not right. It sounds as if it stores different date format constants. Whereas here we are `formatting` something into a different format.